### PR TITLE
fix:Removed extra space from testimonials.js

### LINKED ISSUE #5640: Extra white space
### Description

There are extra spaces between some sections. It looks ugly when there is more space so we can remove those extra spaces. 

### Screenshots

Screenshot from 2023-03-24 23-11-03
Screenshot from 2023-03-24 23-11-13

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yoyoekta**
## Fixes Issue
Closes #5640
## Changes proposed
Removed extra spaces from Testimonials.js

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5648"></a>

### CODE REVIEW

**Review by hirentimbadiya — COMMENTED** [review]
Looks good!!  but why you haven't ticked check boxes ?

**Comment by yoyoekta** [comment]
> Looks good!! but why you haven't ticked check boxes ?

Sorry for the same. I will keep this in mind from now.

File: components/Testimonials.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Link from "./Link";
 
 export default function Testimonials({ data }) {
   return (
-    <div className="bg-white py-24 sm:py-32">
+    <div className="bg-white pt-0 pb-24 sm:pt-0 pb-24">
```
Comment (eddiejaoude) [inline components/Testimonials.js:5]: There is some duplicate and extra white space, try this...

```suggestion
    <div className="bg-white pt-0 pb-24 sm:pt-0">
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution, I added some inline comments

File: components/Testimonials.js (hunk 1 above)
Comment (eddiejaoude) [reply components/Testimonials.js:5]: It looks like the force push revert your latest changes. I will commit my suggestions

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

File: components/Testimonials.js (hunk 1 above)
Comment (yoyoekta) [reply components/Testimonials.js]: > It looks like the force push revert your latest changes. I will commit my suggestions

That means merge commit was better option. Will take care of that in future. Thanks a lot!

Comment (eddiejaoude) [reply components/Testimonials.js]: I very rarely use force push, you made improvements and then in the following commit with a force you removed them

No problem, we are all learning and these things take practice. Next time will be easier, keep going Ekta

**Pull request merged**